/**:
  ros__parameters:
    image_orientation: 0
    # 图像宽度:步进16
    nWidth: 3072
    # 图像高度：步进2
    nHeight: 2048
    # 水平偏移
    nOffsetX: 0
    # 垂直偏移
    nOffsetY: 0
    # 水平镜像
    bReverseX: false
    # 垂直镜像
    bReverseY: false
    # 像素格式
    # 013: 0x0108000A bayer gb8
    # 013: 0x01100012 bayer gb12
    # 016: 0x01080009 bayer rg8
    # 016: 0x01100011 bayer rg12
    nPixelFormat: 0x01100011
    # 水平合并
    nBinningHorizontal: 1
    # 垂直合并
    nBinningVertical: 1
    # 水平下采样
    nDecimationHorizontal: 1
    # 垂直下采样
    nDecimationVertical: 1
    # 采集触发帧率
    nAcquisitionBurstFrameCountValue: 1
    # 设置采集帧率
    fFPS: 404.0000
    # 设置使能采集帧率控制
    bEnableAcqFrameRate: false
    # 曝光时间
    fExposureTime: 8000.0000   #4000.0
    # 增益
    fGain: 19.0000
    # 自动增益
    nGainAuto: 0
    # 黑电平
    nBlackLevelValue: 30
    # 黑电平使能
    bEnableBlackLevel: false
    # 伽马校正
    fGammaValue: 0.7
    # 伽马校正使能
    bEnableGamma: 1

    # Bayer转换质量 0: fast 3: best+
    nBayerCvtQuality: 3

    camera_img_topic: "image"
